scripts/record_and_replay.py

```diff
@@ -21,6 +22,16 @@ def gen_tf_var_list(var_name: str, vals: list) -> str:
     list_str = ','.join(f'"{v}"' for v in vals)
     return f'{var_name} = [{list_str}]'
 
+def execute_cmd_with_text_output(cmd: list, wdir: str, err: str) -> str:
+    try:
+        return check_output(cmd, cwd=wdir, stderr=DEVNULL)
+    except CalledProcessError:
+        print(f"ERROR: {err}")
+        sys.exit(1)
```

Comment:
I suggest rethrowing the exception (with just `throw`) which will print a stacktrace and full exception message.

Reply:
done